### DISCUSSION TITLE
Letter "p" can't be entered into text fields or search box in ReaLearn

### DISCUSSION THREAD

**Discussion by rcaia** [helgoboss/helgobox#1335] [Q&A]
I'm unable to enter 'p' [lowercase only] into any ReaLearn dialog boxes that normally accept text... Upper case P and all other characters appear to act as expected.  Only happens inside ReaLearn.  [Reaper and other plugins are not affected].  v2.16.14, Windows 10, Reaper v7.27.

**Comment by helgoboss** [comment]
Solved in 2.16.15

**Comment by rcaia** [comment]
Thanks much... fix is much appreciated.

Robin


------ Original Message ------
From "Benjamin Klum" ***@***.***>
To "helgoboss/helgobox" ***@***.***>
Cc "RCAIA" ***@***.***>; "Author" ***@***.***>
Date 12/14/2024 12:16:30 AM
Subject Re: [helgoboss/helgobox] Letter "p" can't be entered into text 
fields or search box in ReaLearn (Discussion #1335)
